RSDKv5/RSDK/User/Core/UserAchievements.cpp

```diff
@@ -123,6 +123,7 @@ void RSDK::SKU::ProcessAchievements()
                     InitString(&achievementStrings[0], achievementText.c_str(), 0);
                     String buffer;
                     CopyString(&achievementStrings[1], achievements->GetAchievementName(&buffer, achievementID));
+                    RemoveStorageEntry((void**)&buffer);
```

Comment:
Are these in the original?

Reply:
Doesn't seem like it. However this is necessary here since `buffer` is allocated by `GetAchievementName()`, creating a dangling pointer to the stack.
Couldn't exactly figure out how the original EGS version handles it.